.github/workflows/test-all.yaml

```diff
@@ -153,3 +153,21 @@ jobs:
       - name: Verify minimum rust version — prqlc
         working-directory: crates/prqlc
         run: cargo msrv verify
+
+  test-publish-to-cargo:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 📂 Checkout code
+        uses: actions/checkout@v3
+      - run: ./.github/set_version.sh
+      - run: |
+          published_version=$(curl -sL https://crates.io/api/v1/crates/prql-compiler/versions | jq -r '.versions[].num' | sort -V | tail -n 1)
```

Comment:
Why have this? Aren't they only equal for the actual release commit?

Reply:
> Why have this? Aren't they only equal for the actual release commit?

My understanding is that we can only find out which version has been published to crates.io by contacting crates.io.
I don't think we can check the Git repository to see if it has been successfully published to crates.io or not.

Reply:
Oh, I see, because otherwise it'll fail...